frontend/src/metabase/ui/components/inputs/MultiSelect/MultiSelect.styled.tsx

```diff
@@ -100,8 +105,8 @@ export const getMultiSelectOverrides =
           fontWeight: "bold",
           fontSize: getSize({ size, sizes: theme.fontSizes }),
           borderRadius: theme.radius.xs,
-          color: theme.fn.themeColor("brand"),
-          backgroundColor: theme.fn.themeColor("bg-medium"),
+          color: "var(--mb-color-text-brand)",
```

Comment:
theme the chevron icon
![image](https://github.com/user-attachments/assets/6af889ce-aa71-470e-98c0-85446be9e2bc)